capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -229,15 +327,24 @@ export class SearchUserModal extends Component {
                               </Card.Text>
 
 
-                              {this.state.addedUsers.includes(user) &&
-                                <Button className="my-2"  variant='danger' onClick={() => this.removeUserFromAddedUsers(user)}>
+                              {this.arrayContainsJSONId(this.state.addedUsers, user) ?
+                                <Button className="my-2 w-75" variant='danger' onClick={() => this.removeUserFromAddedUsers(user)}>
                                   Remove
                                 </Button>
+                                :
+                                <Button className="my-2 w-75" onClick={() => this.addUserToAddedUsers(user)}>
+                                  Add
```

Comment:
Is this "add" too ambiguous? I'm guessing this is so it can be used on multiple modals for different purposes but it's slightly unclear

Reply:
Yeah, I will add a prop so you can specify the text.